pkg/tsdb/grafana-pyroscope-datasource/query.go

```diff
@@ -430,15 +437,32 @@ func seriesToDataFrames(resp *SeriesResponse) []*data.Frame {
 
 		valueField := data.NewField(resp.Label, labels, []float64{})
 		valueField.Config = &data.FieldConfig{Unit: resp.Units}
+		fields = append(fields, valueField)
+
+		annotationsField := data.NewField("annotations", nil, []json.RawMessage{})
+		hasAnnotations := false
 
 		for _, point := range series.Points {
 			timeField.Append(time.UnixMilli(point.Timestamp))
 			valueField.Append(point.Value)
+			if len(point.Annotations) > 0 {
```

Comment:
What's the best way to test it locally (in a way that Pyroscope returns some annotations)